Examples/restcomm-olympus/app/src/androidTest/java/org/restcomm/android/olympus/IntegrationTests.java

```diff
@@ -1029,6 +1046,511 @@ public void run() {
     }
 
 
+    /**
+     *
+     *
+     * Push notification Integration Tests
+     *
+     *
+     *
+     */
+    @Test(timeout = TC_TIMEOUT)
+    public void deviceInitializeWithPush_FCMKeyMissing() throws InterruptedException {
+        InstrumentationRegistry.getTargetContext().bindService(new Intent(InstrumentationRegistry.getTargetContext(), RCDevice.class), this, Context.BIND_AUTO_CREATE);
+        await().atMost(SIGNALING_TIMEOUT, TimeUnit.SECONDS).until(fieldIn(this).ofType(boolean.class).andWithName("serviceConnected"), equalTo(true));
+
+        // Get the reference to the service, or you can call public methods on the binder directly.
+        final RCDevice device = binder.getService();
+
+        Handler clientHandler = new Handler(Looper.getMainLooper());
+
+        clientHandler.post(new Runnable() {
+            @Override
+            public void run() {
+                resetPushConfiguration();
```

Comment:
I'd like to avoid messing with internal model state from the TC. Let's discuss if/how we can improve this when you get back

Reply:
done. Removed it.